docs/src/main/paradox/release-notes/10.2.x.md

```diff
@@ -1,14 +1,83 @@
 # 10.2.x Release Notes
 
-## 10.2.0
-
 ### Migration notes
 
-#### Scalatest dependency upgraded to 3.1.0
-
-The Scalatest dependency for akka-http-testkit was upgraded to version 3.1.0. This version is incompatible with previous
-versions. This is relevant for user code if it uses methods from @scaladoc[ScalatestUtils](akka.http.scaladsl.testkit.ScalatestUtils)
-(which are in scope if your test extends from @scaladoc[ScalaTestRouteTest](akka.http.scaladsl.testkit.ScalaTestRouteTest)).
-In this case, the project itself needs to update to use Scalatest >= 3.1.0.
+See the @ref[Migration guide](../migration-guide/migration-guide-10.2.x.md).
 
 ### Changes since 10.1.11
+
+This page currently contains the changes between 10.1.11 and the first
+milestone release of 10.2.x, 10.2.0-M1. This list will be amended when
+we release 10.2.0 proper.
+
```

Comment:
We should feature a few things here:

 * proper 2.6 support
 * support for message attributes
 * link to implicit changes in routing API

Reply:
I guess we could duplicate the list of highlights from the announcement?

Reply:
Yes, exactly, only saw these afterwards.